### PR TITLE
Add large event history fixture

### PR DESCRIPTION
**Pull request by sw-yx**
## What was changed
add Large Event History Fixture

## Why?
needed it for https://github.com/temporalio/web/issues/300

### CODE REVIEW

**Comment by feedmeapples** [comment]
very nice, that's gonna be helpful.

btw there is a way to address the original issue itself too, i'll get to it sometime soon https://github.com/temporalio/web/issues/300

**Comment by sw-yx** [comment]
@feedmeapples btw i did this half way and then got distracted.. i dont think it works properly yet - if you want to take over please be my guest, as i think theres a chance this becomes one of my abandoned projects with the other stuff i have goign on

**Review by feedmeapples — APPROVED** [review]

**Comment by feedmeapples** [comment]
> @feedmeapples btw i did this half way and then got distracted.. i dont think it works properly yet - if you want to take over please be my guest, as i think theres a chance this becomes one of my abandoned projects with the other stuff i have goign on

finished the fixture 
cc @stevekinney

File: temporal-fixtures/largepayload/README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-This fixuture starts several workflows `(NumberOfWorkflows)` with payloads of size 1mb `(PayloadSize)`. These payloads include: memo, activity input, activity result.
+This fixuure starts several workflows `(NumberOfWorkflows)` with payloads of size 1mb `(PayloadSize)`. These payloads include:
```
Comment (feedmeapples) [inline temporal-fixtures/largepayload/README.md:1]:
```suggestion
This fixture starts several workflows `(NumberOfWorkflows)` with payloads of size 1mb `(PayloadSize)`. These payloads include: 
```

File: temporal-fixtures/large-event-history/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+This fixture starts a single workflow that adds a lot of event histories and then fails.
```
Comment (feedmeapples) [inline temporal-fixtures/large-event-history/README.md:1]:
```suggestion
This fixture starts a single workflow that adds a lot of event histories `(LengthOfHistory)` and then fails `(WillFailOrNot)`.
```

**Pull request merged**